assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -124,7 +125,7 @@ function MediaPane(props) {
    */
   const onSelect = (mediaPickerEl) => {
     const resource = getResourceFromMediaPicker(mediaPickerEl);
-    insertMediaElement(resource);
+    insertMediaElement(resource, mediaPickerEl.sizes.medium.url);
```

Comment:
Is this the exact URL that is used in the media Picker? If so, can you add a comment explaining that?